paper/paper.md

```diff
@@ -30,40 +30,40 @@ bibliography: paper.bib
+[ThermoFun](https://thermohub.org/thermofun) is an open source library that simplifies the calculation and retrieval of standard thermodynamic properties of chemical substances, compounds, and reactions among them at a range of temperatures and pressures. The library is developed in C++ for performance, but it also has a Python API for broader and more convenient usage. It employs a variety of thermodynamic models and equations of state for solid, aqueous, surface, gaseous, and molten substances, and their reactions with input properties and parameters from various thermodynamic datasets that are collected and curated in the in [ThermoHub](https://thermohub.org/thermohub) database. The library can be used as a standalone code for searching and tabulating thermodynamic properties or linked to other modeling codes that require standard thermodynamic data as input. It offers the flexibility to use different thermodynamic datasets, including custom datasets and datasets retrieved from the online ThermoHub database, and to choose the most suitable models for different classes of substances necessary in various modeling applications. It can serve as a common source of thermodynamic models for standard properties of substances and reactions that can be easily integrated and combined, significantly improving the modeling capabilities for diverse (geo)chemical systems and over wide ranges of conditions.
```

Comment:
Should the word *standard* be used before *thermodynamic models and equations of state*?

Reply:
Let's leave this in general - "thermodynamic models". No change needed here. Thanks for the explanation, Dmitrii.